src/NUnitFramework/framework/Constraints/ComparisonConstraint.cs

```diff
@@ -143,11 +142,43 @@ public ComparisonConstraint Percent
             get
             {
                 _tolerance = _tolerance.Percent;
-                Description += " percent";
                 return this;
             }
         }
 
         #endregion
+        
+        #region Protected Methods
+        
+        /// <summary>
+        /// Provides standard description of what the constraint tests
+        /// based on comparison text.
+        /// </summary>
+        /// <param name="comparisonText">Describes, what constraint tests, throws <see cref="ArgumentNullException"/>
+        /// if null</param>
+        /// <exception cref="ArgumentNullException">Is thrown when null passed to a method</exception>
+        protected string DefaultDescription(string comparisonText)
+        {
+            if (comparisonText == null)
+                throw new ArgumentNullException(nameof(comparisonText), "Comparison text can not be null");
+            
+            System.Text.StringBuilder sb = new System.Text.StringBuilder(comparisonText);
+            sb.Append(MsgUtils.FormatValue(_expected));
```

Comment:
💭 Side note: we should think about having MsgUtils write directly into a StringBuilder.

Reply:
Didn't quite get the idea.
If you are talking about adding a method to `MsgUtils`, which interacts with `StringBuilder` directly, I can work on it after closing that issue.

Reply:
Sorry, I should have @'ed the team. This is an idea for the future that I didn't mean to ask you to do. (Unless you feel like it.)